README.md

```diff
@@ -1 +1,58 @@
-# JavaScript-capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# JavaScript group capstone
+
+
+In this project I have used linter for HTML and CSS and Javascript, to make sure there are no linter errors on the code.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+
+## Live Demo (if available)
+
+[Demo](https://github.com/tarek102/JavaScript-capstone//dist/)
+
+
+## Getting Started
+
+To successfully utilize this project first of all you need to clone or download the repository. Next, open the index.html file to check the website’s main page. If you want to add or edit the files you can do that by simply opening the file folder in VS code.
+
```

Comment:
- [ ] Great job adding the `Getting Started` section 💯. However,  I suggest adding more instructions about how to run your project, so I suggest adding `npm install` to install all dependencies, `npm run start` to run the project, and  `npm run test` to run the test 👌